installation/resources/README.md

```diff
@@ -0,0 +1,36 @@
+# Control Plane Resources
+
+## Installation CRs
+
+The following table lists all the Installation custom resource files for Control Plane:
+
+| File                                      | Description                                                          |
+|-------------------------------------------|----------------------------------------------------------------------|
+| `installer-cr-kyma-dependencies.yaml`     | Kyma components that are required for Control Plane installation.    |
+| `installer-cr-compass-dependencies.yaml`  | Compass components that are required for Control Plane installation. |
+| `installer-cr.yaml.tpl`                   | Components installed by the Control Plane Installer.                 |
+
+## KYMA_VERSION file
+
+`KYMA_VERSION` is the file specifying the version of Kyma to use during the installation.
+
+### Possible values
+
+| Value                     | Example Value         | Explanation               |
+|---------------------------|-----------------------|---------------------------|
+| `master`                  | `master`              | Latest master version.    |
```

Comment:
```suggestion
| `master`                  | `master`              | Latest master version    |
```

Reply:
I'd remove the dot since this is not a full sentence. Please apply in other rows as well.